### PR TITLE
Add WanSync docs [CN-369]

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+= Syncronizing WAN replicated data
+:description: pass:q[WAN Syncronization allows you to overcome this out-of-sync issues with the WAN replicated data. ]
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:2]:
```suggestion
:description: pass:q[WAN Syncronization allows you to overcome out-of-sync issues with the WAN replicated data. ]
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+:page-enterprise: true
+
+{description}
+
+See xref:hazelcast:wan:advanced-features#synchronizing-wan-clusters.adoc[Synchronizing WAN Clusters] for more detailed information about the configuration and behavior of WAN replication in Hazelcast.
+
+== Prerequisites for WAN Sync
+
+Before you create and configure a `WanSync` resource, you need to create the following:
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:11]:
```suggestion
Before you create and configure a `WanSync` resource, you must create the following:
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+- At least one `Hazelcast` resource and one `Map` resource for the source Hazelcast cluster.
+- A target Hazelcast cluster that is accessible from the source cluster.      
+
+== Configuring the WanSync Resource
+
+To configure WAN Syncronization, you need to create `WanSync` resource. For more detailed information, see the xref:api-ref.adoc#wansyncspec[API Reference].
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:18]:
```suggestion
To configure WAN Syncronization, you must create `WanSync` resource. For more detailed information, see the xref:api-ref.adoc#wansyncspec[API Reference].
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+There are two ways to create a `WanSync` resource. The first one is by spcifying the existing `WanReplication` name. This way, the `WanSync` will reuse the existing and running WAN replication proccess.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:20]:
```suggestion
There are two ways to create a `WanSync` resource. The first one is by specifying the existing `WanReplication` name. This means that the `WanSync` reuses the existing and running WAN replication process.
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+.Example of WanSync using the existing WanReplication
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/wan-sync-from-replication.yaml[]
+----
+
+NOTE: For using this appoach, you must make sure that the `WanReplication` resource is succesully created.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:28]:
```suggestion
NOTE: For this approach, you must make sure that the `WanReplication` resource is successfully created.
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+Another apporach is to specify the WAN replication configuration directly in the `WanSync` resource. This way, the operator will create a new `WanReplication` resource before running the WAN syncronization procces. The created `WanReplication` will be deleter when the correspondig `WanSync` is deleted.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:30]:
```suggestion
Another approach is to specify the WAN replication configuration directly in the `WanSync` resource. This means that the operator creates a new `WanReplication` resource before running the WAN syncronization process. The created `WanReplication` is deleted when the corresponding `WanSync` is deleted.
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+.Example of WanSync with the new WanReplication configuration
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/wan-sync-config.yaml[]
+----
+
+== Checking the Status of a WAN Sync
+
+After creating the `WanSync` resource, run the following command to find out the WAN syncronization status:
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:40]:
```suggestion
After creating the `WanSync` resource, run the following command to see the WAN syncronization status:
```

File: docs/modules/ROOT/pages/wan-sync.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+[source,shell]
+----
+kubectl get wansync wansync-sample
+----
+
+You should see the following output:
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/wan-sync.adoc:47]:
```suggestion
The output is similar to the following:
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Review by cagric0 — APPROVED** [review]

**Review by Serdaro — APPROVED** [review]

**Review by semihbkgr — APPROVED** [review]

File: docs/modules/ROOT/examples/wan-sync-full.yaml [hunk 9]
```diff
@@ -0,0 +1,6 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: WanSync
+metadata:
+  name: wansync-sample
+spec:
+  wanReplicationName: wan-replication-sample
```
Comment (cheels) [inline docs/modules/ROOT/examples/wan-sync-full.yaml:6]: Here should be **wanReplicationResourceName**

**Review by cheels — APPROVED** [review]

**Pull request merged**